### ISSUE TITLE
Dedicated Server Installation?

### ISSUE THREAD

**Issue by SubSonix** [dyoung522/valheim-plus-installer#16]
I just tried this and it won't install to the directory as the .exe has a different name. I changed it manually before and after installation to trick it into installing, just hoping you could allow it to install to the dedicated server installation by default too?

**Comment by dyoung522** [comment]
Yup, I've had several requests for this... added to the roadmap. Thanks for the suggestion!

**Comment by SubSonix** [comment]
Thank you for the application :)
